### ISSUE TITLE
Question about the pgd.py file

### ISSUE THREAD

**Issue by ahmadajal** [locuslab/projected_sinkhorn#4]
Hello, 

I was checking your code and I had a question about the pgd.py file. When you want to project $X_$ on to the Wasserstein ball, shouldn't you have ` X_ .data[~err] = projected_sinkhorn(...)` in line 50? I mean we do not want to differentiate the projection, right?

Thanks a lot,
-Ahmad

**Comment by riceric22** [comment]
Hi Ahmad! 

The variable assignment here is being done with the scope of `with torch.no_grad()` on L33 and so gradients are already disabled. So the projection will not be differentiated. 

~Eric

**Comment by ahmadajal** [comment]
Ah, yes. My bad.
many thanks for your answer.

-Ahmad
